lib/oli_web/live/projects/table.ex

```diff
@@ -25,12 +25,19 @@ defmodule OliWeb.Projects.Table do
           <%= th(assigns, "Title", @sort_by, @sort_order, "title") %>
           <%= th(assigns, "Created", @sort_by, @sort_order, "created") %>
           <%= th(assigns, "Collaborators", @sort_by, @sort_order, "author") %>
+          <%= th(assigns, "Status", @sort_by, @sort_order, "status") %>
```

Comment:
The `status` column should only be displayed if the user is an admin user.  Otherwise, all non-admin users will simply see this `status` column that only ever says "Active"

Reply:
Removed  status column for non-admins